artifacts_test.py

```diff
@@ -475,34 +513,63 @@ def test_scylla_service(self):
         if self.params.get('client_encrypt') and SkipPerIssues("https://github.com/scylladb/field-engineering/issues/2280", self.params):
             self.log.info("Scylla Doctor test is skipped for encrypted environment due to issue field-engineering#2280")
+            self.actions_log.info(
+                "Scylla Doctor test is skipped for encrypted environment due to issue field-engineering#2280")
             return
 
         if self.db_cluster.nodes[0].is_nonroot_install and \
                 SkipPerIssues("https://github.com/scylladb/scylla-cluster-tests/issues/10540", self.params):
             self.log.info("Scylla Doctor test is skipped for non-root test due to issue field-engineering#2254. ")
+            self.actions_log.info(
+                "Scylla Doctor test is skipped for non-root test due to issue field-engineering#2254.")
             return
 
         if self.node.parent_cluster.cluster_backend == "docker":
             self.log.info("Scylla Doctor check in SCT isn't yet support for docker backend")
+            self.actions_log.info("Scylla Doctor check in SCT isn't yet support for docker backend")
             return
 
         for node in self.db_cluster.nodes:
             scylla_doctor = ScyllaDoctor(node, self.test_config, bool(self.params.get('unified_package')))
+            self.actions_log.info("Installing Scylla Doctor", target=node.name)
+            scylla_doctor.install_scylla_doctor()
+            self.actions_log.info("Scylla Doctor is installed", target=node.name)
+            scylla_doctor.argus_collect_sd_package()
+            self.actions_log.info("Running Scylla Doctor", target=node.name)
+            scylla_doctor.run_scylla_doctor_and_collect_results()
+            self.actions_log.info("Scylla Doctor is finished", target=node.name)
+            self.actions_log.info("Analyzing Scylla Doctor results", target=node.name)
+            scylla_doctor.analyze_vitals()
+            scylla_doctor.analyze_and_verify_results()
             scylla_doctor.install_scylla_doctor()
+            self.actions_log.info("Scylla Doctor is installed", target=node.name)
             scylla_doctor.argus_collect_sd_package()
+            self.actions_log.info("Running Scylla Doctor", target=node.name)
+            scylla_doctor.run_scylla_doctor_and_collect_results()
+            self.actions_log.info("Scylla Doctor is finished", target=node.name)
+            self.actions_log.info("Analyzing Scylla Doctor results", target=node.name)
+            scylla_doctor.analyze_vitals()
             scylla_doctor.run_scylla_doctor_and_collect_results()
+            self.actions_log.info("Scylla Doctor is finished", target=node.name)
+            self.actions_log.info("Analyzing Scylla Doctor results", target=node.name)
             scylla_doctor.analyze_vitals()
             scylla_doctor.analyze_and_verify_results()
+            self.actions_log.info("Scylla Doctor test is completed", target=node.name)
```

Comment:
I really dislike this usage, we need to figure out a way how to make this look better. One way would be to hide it inside the functions themselves (at least most of the logs).

Reply:
I don't like putting this into utility function themselves, we end up with too much of it.

also the level of verbosity in this case is way too much, and it would be helpful it's it's a context manger like event or log to node context managers

Reply:
I reduced the level of verbosity and used context mgr for that - now looks better